packages/docs/src/pages/components/paragraph.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Paragraph
+---
+
+# Paragraph
+
+Primitive typographic component for text blocks, defaults to `<p>`, and without margin.
```

Comment:
Nitpick. There's something not right with this sentence.

Reply:
Something like my frenglish, perhaps? 😁